Update opendronemap.md (#1261)
CSCfi/csc-user-guide: docs/apps/opendronemap.md

__OpenDroneMap__ is available in Puhti with following versions:

* 2.8.3
* 2.5.7

## Usage
OpenDroneMap is available in Puhti as a [Singularity](../computing/containers/run-existing.md) container

> 1) Copy your images first to Puhti. The folder name needs to ènd with `code/images`, for example `/scratch/project_2000XXX/odm/code/images`.

Reviewer: ```suggestion
To run OpenDroneMap: 
1) Copy your images to Puhti. OpenDroneMap requires the folder names to end with `code/images`, for example `/scratch/project_2000XXX/odm/code/images`.
```

> 2) Submit your OpenDroneMap batch job. 

Reviewer: ```suggestion
2) Write a batch job script (see below)
3) Submit your OpenDroneMap batch job. 
```

> * According to our tests a project with ~300 images is optimal to run with 8-12 CPU cores, adjust the number of CPUs on the `--cpus-per-task` line. OpenDroneMap can use only one computing node in processing which means maximum of 40 CPU cores.

Reviewer: ```suggestion
* According to our tests, a project with ~300 images is optimal to run with 8-12 CPU cores; adjust the number of CPUs on the `--cpus-per-task` line. OpenDroneMap can use only one computing node in processing which means that it is limited to a maximum of 40 CPU cores per job.
```

* `--project-path` - the place where images are stored, without the `code/images` part.
* `--max-concurrency` - the number of threads used in several steps of ODM processing, here set to same number as reserved cores. 
* It is possible to add [additional arguments](https://docs.opendronemap.org/arguments/) to the end of the command. 
* If your images cover a very large area, see [ODM Splitting Large Datasets documentation](https://docs.opendronemap.org/large/)

> 

Reviewer: ```suggestion
Below you can find an example batch job script. Adjust `--account`, `--cpus-per-task`, `--time` and `--mem-per-cpu` to your needs.
```
